docs/BuildBrainFlow.rst

```diff
@@ -149,6 +149,44 @@ Rust
         cd brainflow
         cargo build --features generate_binding
 
+Swift
+-------
+
+You can build the Swift bindings for BrainFlow using Xcode. Before that you need to compile C/C++ code :ref:`compilation-label` and ensure that native libraries are properly placed. Keep in mind that currently it supports only MacOS.
+
+To compile your Swift app with the BrainFlow bindings, first copy all of the .swift files from swift_package/BrainFlow/Sources/BrainFlowBindings into your Xcode project:
```

Comment:
copypasting files is not good, I actually thought that its smth like package. e.g. https://www.swiftyplace.com/blog/modular-code-with-swift-package-manager 
Can also publish it on github packages later on. BrainFlowCI should use it as a package also

Reply:
Creating the package is an important feature, but it is also a challenge and will take a while. I've tried several times over the years but failed each time. I will most likely have to open a help ticket with Apple Developer Support. I cannot promise a delivery date for this feature.

Reply:
https://developer.apple.com/documentation/xcode/creating-a-standalone-swift-package-with-xcode here is a good guide how to create it, I believe the 1st step should be to separate BrainFlowCI and BrainFlowBinding to different projects, create a package like described in the article and after that try to use this package in CI apps